src/core/server/ui_settings/settings/theme.test.ts

```diff
@@ -58,12 +58,12 @@ describe('theme settings', () => {
     const validate = getValidationFn(themeSettings['theme:version']);
 
     it('should only accept valid values', () => {
-      expect(() => validate('v7')).not.toThrow();
-      expect(() => validate('v8 (beta)')).not.toThrow();
```

Comment:
Can we keep these and expect them to throw?

Reply:
same reason as above, It will validate based on the schema we defined